### PR TITLE
issue fix #9581: heading strucutre fails WCAG 2.2AA- skipped heading

### LINKED ISSUE #9581: [BUG] Heading structure fails WCAG 2.2 AA - Skipped heading level
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

production on biodrop.io

### Version of BioDrop (for example "v1.2.3")

v2.80.1

### Description

Checking the source code with WAVE and manually, it is shown that the heading structure skips one level.

**Example:** EddieJaoude profile https://www.biodrop.io/eddiejaoude

The first heading that appears is
`<h1 class="flex text-3xl font-bold gap-1">Eddie Jaoude | EddieHub<!-- --> <!-- -->(he/him)</h1>`

The next heading that appears is
`<h3 class="ml-2 mt-2 text-lg font-medium leading-6 dark:text-primary-low text-primary-high">Socials Primary</h3>`

`<h2></h2>` is missing. This is misleading for screen reader users.

### Screenshots

Wave results for skipped heading level

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

I would turn all the `<h3>` of this page into a `<h2>`.
Since the heading tags have a styling set using tailwind, nothing changes in their appearance.

**Shoutout:** This problem is already prevented in the footer section through
`<h2 id="footer-heading" class="sr-only">Footer</h2>`

Good job 🚀

### PR DESCRIPTION
**Pull request by berryjen**
## Fixes Issue
closes #9581

## Changes proposed
modified all the h3 tags in all the files into a h2.
modified all the h3 tags 

## Check List (Check all the applicable boxes) 

- [ X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X] All new and existing tests passed.
- [X ] This PR does not contain plagiarized content.

- [X ] The title of my pull request is a short description of the requested changes.

## Screenshots

 

 There are no longer any skipped heading warnings under the structure tab when using WAVE to inspect accessibility. 

Screenshot 2023-10-26 at 8 37 15 PM

## Note to reviewers

hopefully the screenshot appears!

### CODE REVIEW

File: tests/data/_test-profile-user-6.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
       "icon": "FaLink"
     },
     {
-      "group": "Socials Primary",
+      "group": "",
```
Comment (sital002) [inline tests/data/_test-profile-user-6.json:20]: Can you explain why are you removing this

**Review by sital002 — CHANGES_REQUESTED** [review]
The ```package-lock.json``` file shouldn't be changed unless you are installing a new package.
Please revert those changes. Here is the reference video for how to revert ```package-lock.json``` file https://www.youtube.com/watch?v=jHkIXYVpMTo&ab_channel=EddieJaoude

**Comment by berryjen** [comment]
I had 2 merge conflicts trying to push my commit to my local branch, and VSC wasn't able to merge the conflict on its own so I had to report to doing that in order to push my code changes.

**Comment by berryjen** [comment]
Is there a way to debug the merge conflict without reinstalling npm? If so, would you be able to help me out? TIA!

**Comment by eddiejaoude** [comment]
> Is there a way to debug the merge conflict without reinstalling npm? If so, would you be able to help me out? TIA!

You probably had merge conflicts because you are making changes in the `main` branch. You can still remove the changes in the lock file back to the `upstream` (original repo not your fork) as Sital suggested - because you haven't made changes to the `package.json`. To reduce these types of conflicts in the future use any branch other than `main`.

**Comment by berryjen** [comment]
> > Is there a way to debug the merge conflict without reinstalling npm? If so, would you be able to help me out? TIA!
> 
> You probably had merge conflicts because you are making changes in the `main` branch. You can still remove the changes in the lock file back to the `upstream` (original repo not your fork) as Sital suggested - because you haven't made changes to the `package.json`. To reduce these types of conflicts in the future use any branch other than `main`.

I'm still confused...i was comparing the main branch of my repo to the` main `branch of eddiehub....the local `main` branch on my GH is the branch i was working on....
Do i delete the changes on my `lock json` file, copy& paste the original repo `lock json` file? 
when you mention to use any other branch other than `main`....but there is only 1 branch on my own repo? Or do you mean eddiehub repo? >"<

**Comment by akash19coder** [comment]
> when you mention to use any other branch other than `main`....but there is only 1 branch on my own repo? Or do you mean eddiehub repo? >
Whenever we are working on an issue, we should create a new branch other than `main` with `git branch <branch-name> command and switch to that branch with `git switch <branch-name>` and make all the changes there related to issue because it prevents the conflicts like we faced earlier with lock-json files

**Comment by eddiejaoude** [comment]
> I'm still confused...i was comparing the main branch of my repo to the`main`branch of eddiehub....the local `main` branch on my GH is the branch i was working on.... Do i delete the changes on my `lock json` file, copy& paste the original repo `lock json` file? when you mention to use any other branch other than `main`....but there is only 1 branch on my own repo? Or do you mean eddiehub repo? >"<

You can revert the changes to your lock file in a few different ways. I think using the `git checkout` command is probably best here with `git checkout main -- package-lock.json`, I made a video on this https://www.youtube.com/watch?v=jHkIXYVpMTo

On your repo, the `main` branch should not contain any of your changes directly from your commits. After your PR is merged in the original repo (upstream) you sync your main branch (in your fork) with the original repo and then your changes will appear in your `main` branch.

**Comment by berryjen** [comment]
> `main`

so does that mean in the future, i need to create 2 branches? One is called main, and the other <branch name> , where i will be pushing my commits to? The PR will be raised on <branch name> branch, which then gets merged with the original repo <eddieHub main>? I then synch my <main>  branch with the <eddieHub main>, which means that i'll have the latest version of all the changes made to <eddieHub main>? 
Sorry for the continuous questions, but this Git stuff really is so confusing, esp. in a commercial setting...I do apologize for my lack of knowledge on it. Believe me I've watched YT tutorials, but without actually trying it out, I never know what to do @__@

**Comment by berryjen** [comment]
> > I'm still confused...i was comparing the main branch of my repo to the`main`branch of eddiehub....the local `main` branch on my GH is the branch i was working on.... Do i delete the changes on my `lock json` file, copy& paste the original repo `lock json` file? when you mention to use any other branch other than `main`....but there is only 1 branch on my own repo? Or do you mean eddiehub repo? >"<
> 
> You can revert the changes to your lock file in a few different ways. I think using the `git checkout` command is probably best here with `git checkout main -- package-lock.json`, I made a video on this https://www.youtube.com/watch?v=jHkIXYVpMTo
> 
> On your repo, the `main` branch should not contain any of your changes directly from your commits. After your PR is merged in the original repo (upstream) you sync your main branch (in your fork) with the original repo and then your changes will appear in your `main` branch.

so does that mean in the future, i need to create 2 branches? One is called main, and the other <branch name> , where i will be pushing my commits to? The PR will be raised on <branch name> branch, which then gets merged with the original repo <eddieHub main>? I then synch my <main>  branch with the <eddieHub main>, which means that i'll have the latest version of all the changes made to <eddieHub main>? Is this mental model correct?
Sorry for the continuous questions, but this Git stuff really is so confusing, esp. in a commercial setting...I do apologize for my lack of knowledge on it. Believe me I've watched YT tutorials, but without actually trying it out, my mental schema in my head is always different from reality

**Comment by eddiejaoude** [comment]
It is confusing, you are right the best way to understand is to try it. `main` will be created on your fork repo already, you should always keep this up to date with the original repo and then create your branches from there

**Comment by berryjen** [comment]
additionally, i did `git status`, which displays theres nothing to commit....im unsure if the reverted changes has been committed to my local branch?!
Screenshot 2023-10-31 at 11 27 26 PM

**Comment by eddiejaoude** [comment]
> additionally, i did git status, which displays theres nothing to commit....im unsure if the reverted changes has been committed to my local branch?!

sorry, it won't work because you made changes in the main branch. Let me try to think of another way

**Comment by berryjen** [comment]
Ahh ok..looks like i made a big oppose 🥲

**Comment by eddiejaoude** [comment]
I can try to fix via a temporary branch - I will take a look tomorrow

**Comment by Abhishek-90** [comment]
> [Screenshot 2023-10-31 at 8 16 07 PM](https://github.com/EddieHubCommunity/BioDrop/assets/73888356/f0b00f3b-c757-464b-bf15-22e3352dc5ad) Nothing appeared on the terminal...is this correct?!

I would say to use `git bash` rather than VSC's inbuilt terminal for git related operations. It will give you a clear picture.

**Comment by eddiejaoude** [comment]
I fixed the lock file changes in a temporary PR here https://github.com/EddieHubCommunity/BioDrop/pull/9719

**Comment by berryjen** [comment]
@eddiejaoude sorry for the late reply, just finished midterms as of yesterday, so spent all last week and early this week studying...@__@ the solution was that you moved #9581 onto a separate branch and then merged the `new` branch onto `eddiehub main`?

**Pull request merged**